src/python/WMCore/Services/DBS/DBS3Reader.py

```diff
@@ -724,18 +728,20 @@ def listDatasetLocation(self, datasetName):
             raise DBSReaderError(msg)
 
         if not blocksInfo:  # no data location from dbs
-            return list()
+            return list(locations)
```

Comment:
Isn't that a minor change of behavior? Before the list returned has been empty.

Reply:
It will still be an empty list. `locations` is an empty set here, so we simply cast it to list. I made this change just to keep it more consistent with the same return at the very end of this method.

Reply:
thanks @amaltaro, that makes sense.